bsip-0061.md

```diff
@@ -0,0 +1,53 @@
+    BSIP: 0061
+    Title: Fix locked accounts with circular dependencies 
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-22-02
+    Updated: 
+    Discussion: https://github.com/bitshares/bsips/issues/94
+    Worker:
+
+
+# Abstract
+BitShares allow to change permissions of account A to account A. See *Cycles* paragraph of https://bitshares.org/technology/dynamic-account-permissions for details. More information can be found here: https://steemit.com/blockchain/@hipster/sad-story-how-i-lost-bitshares-account
+
+# Motivation
```

Comment:
From BSIP1: The motivation is critical for BSIPs that want to change the BitShares protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the BSIP solves. BSIP submissions without sufficient motivation may be rejected outright.

Reply:
Done.